man/sd_bus_creds_new_from_pid.xml

```diff
@@ -137,34 +169,63 @@
     <constant>SD_BUS_CREDS_SELINUX_CONTEXT</constant>,
     <constant>SD_BUS_CREDS_AUDIT_SESSION_ID</constant>,
     <constant>SD_BUS_CREDS_AUDIT_LOGIN_UID</constant>,
+    <constant>SD_BUS_CREDS_TTY</constant>,
     <constant>SD_BUS_CREDS_UNIQUE_NAME</constant>,
     <constant>SD_BUS_CREDS_WELL_KNOWN_NAMES</constant>,
-    or <constant>_SD_BUS_CREDS_ALL</constant> to indicate
-    all known fields.</para>
+    <constant>SD_BUS_CREDS_DESCRIPTION</constant>. Use the special
+    value <constant>_SD_BUS_CREDS_ALL</constant> to request all
+    supported fields. The <constant>SD_BUS_CREDS_AUGMENT</constant>
+    may not be ORed into the mask for invocations of
+    <function>sd_bus_creds_new_from_pid()</function>.</para>
 
     <para>Fields can be retrieved from the credentials object using
     <citerefentry><refentrytitle>sd_bus_creds_get_pid</refentrytitle><manvolnum>3</manvolnum></citerefentry>
     and other functions which correspond directly to the constants
     listed above.</para>
 
-    <para>A mask of fields which were actually successfully set
-    (acquired from <filename>/proc</filename>, etc.) can be retrieved
-    with <function>sd_bus_creds_get_mask()</function>. If the
-    credentials object was created with
+    <para>A mask of fields which were actually successfully retrieved
+    can be retrieved with
+    <function>sd_bus_creds_get_mask()</function>. If the credentials
+    object was created with
     <function>sd_bus_creds_new_from_pid()</function>, this will be a
     subset of fields requested in <parameter>creds_mask</parameter>.
     </para>
 
-    <para><function>sd_bus_creds_ref</function> creates a new
+    <para>Similar to <function>sd_bus_creds_get_mask()</function> the
+    function <function>sd_bus_creds_get_augmented_mask()</function>
+    returns a bitmask of field constants. The mask indicates which
+    credential fields have been retrieved in a non-atomic fashion. For
+    credential objects created via
+    <function>sd_bus_creds_new_from_pid()</function> this mask will be
+    identical to the mask returned by
+    <function>sd_bus_creds_get_mask()</function>. However, for
+    credential objects retrieved via
+    <function>sd_bus_get_name_creds()</function> this mask will be set
+    for the credential fields that could not be determined atomically,
+    and which were later added by reading augmenting credential data
+    from <filename>/proc</filename>. Similar, for credential objects
```

Comment:
Actually, it's the other way round: sd_bus_creds_augmented_mask() is always a subset (or identical) to the mask returned by sd_bus_creds_mask(). The latter hence returns every field available in the object, the former just the fields that are augmented. 

I added some extra wording now to clarify that.